### ISSUE TITLE
Jitsi permanently kicks out one member

### ISSUE THREAD

**Issue by wiesener** [jitsi/docker-jitsi-meet#1705]
Hi,

we are using self hosted docker-jitsi. We are all using it via Chome on MacOS.

One college has always the issue that he periodically is kicked out / freezed after 1-5 min. When he returns to the meeting I am getting the following error message in my js console:

```
JitsiMeetJS.ts:505 2024-01-12T10:19:46.216Z [JitsiMeetJS.ts] <Object.getGlobalOnErrorHandler>:  UnhandledError: Jingle error: {"reason":"timeout","session":"JingleSessionPC[session=P2P,initiator=true,sid=479497cadce7]"} Script: null Line: null Column: null StackTrace:  Error: Jingle error: {"reason":"timeout","session":"JingleSessionPC[session=P2P,initiator=true,sid=479497cadce7]"}
    at https://ssnjitsi.sensorstim.com/libs/lib-jitsi-meet.min.js?v=7499:2:794910
    at F.TimedHandler.handler (https://ssnjitsi.sensorstim.com/libs/lib-jitsi-meet.min.js?v=7499:2:507344)
    at F.TimedHandler.run (https://ssnjitsi.sensorstim.com/libs/lib-jitsi-meet.min.js?v=7499:2:502751)
    at F.Connection._onIdle (https://ssnjitsi.sensorstim.com/libs/lib-jitsi-meet.min.js?v=7499:2:518660)
    at https://ssnjitsi.sensorstim.com/libs/lib-jitsi-meet.min.js?v=7499:2:518819
```

Furthermore, he sees the following error message:
![grafik](https://github.com/jitsi/docker-jitsi-meet/assets/12429559/e8196898-1893-4133-9859-2c0e16693616)

All other colleges do not have this issue.

**Comment by saghul** [comment]
> {"reason":"timeout"

Any chance they are on some different network that has some problems?

**Comment by wiesener** [comment]
Yes, all participants are in a different network. But our college already tried to switch to LTE router and check if this solves the issue but the same happens. In our company network it does not occur. And this is the network where the jitsi docker server is running. 

And the timeout is error is coming on my instance after I realize that my colleges session in jitsi is freezing. So the error message is not from the network where it freezes.

**Comment by saghul** [comment]
Please share the full logs for the participant that fails.

I do still suspect a network problem though.

**Comment by saghul** [comment]
Yes.

**Comment by wiesener** [comment]
Okay, in the mean-time we found a solution. The issue only appears, when during the jitsi session inside (chrome, opera etc.) at the same time Safari is open. If we close Safari (not only background) and run jitsi in chrome it works.

**Comment by saghul** [comment]
That's very unusual. What version of the Docker setup are you using?

**Comment by wiesener** [comment]
Does the docker-compose.yml help? 
[docker-compose.txt](https://github.com/jitsi/docker-jitsi-meet/files/14050129/docker-compose.txt)

**Comment by saghul** [comment]
It doesn't, because you are using images like `<image>:stable` rather ran using the yaml file out of a release. Can you try that please? Otherwise you might be missing some env vars or something....

**Comment by wiesener** [comment]
The ENV files include some sensible informations which we would not like to share.

**Comment by saghul** [comment]
Then take this file: https://github.com/jitsi/docker-jitsi-meet/blob/stable-9111/docker-compose.yml and adapt it to your vars. When you run it it will pull the latest release.

**Comment by wiesener** [comment]
Okay, our admin used the docker-compose.yml from the stable blob. But the issues were the same and we had some Audio connection trouble. So I would assume, that the issue persists.
